core/blockly.js

```diff
@@ -222,6 +226,13 @@ Blockly.onKeyDown_ = function(e) {
       Blockly.mainWorkspace.undo(e.shiftKey);
     }
   }
+  if (deleteBlock) {
```

Comment:
I think this is unnecessary.  As I recall, the divergence in this function is deliberate.